src/main/java/org/drtshock/Potato.java

```diff
@@ -53,6 +53,21 @@ public boolean isPutIntoOven() {
         }
     }
 
+    public boolean isOldPotato() {
```

Comment:
Lots of overhead for checking if this is an old potato. If you're going to return false no matter what, don't query Google! Also, `return false;` in the catch and as the last statement is redundant. You really only need one.